### PR TITLE
v8.5.3 make owner ldap configurable

### PR DESCRIPTION
**Pull request by tpurschke**
- allow for owner groups being external ldap groups (non-editable)
- fix for errors in ip data import: do not delete areas based on name differences but only compare IdStrings

### CODE REVIEW

File: roles/database/files/sql/idempotent/fworch-texts.sql [hunk 1]
```diff
@@ -2237,6 +2237,12 @@ INSERT INTO txt VALUES ('importAppDataPath',    'German', 	'Pfad und Name von Ap
 INSERT INTO txt VALUES ('importAppDataPath',    'English', 	'Path and Name of App data import (without ending)');
 INSERT INTO txt VALUES ('importAppDataSleepTime','German', 	'Import Appdaten-Intervall (in Stunden)');
 INSERT INTO txt VALUES ('importAppDataSleepTime','English', 'Import App data sleep time (in hours)');
+INSERT INTO txt VALUES ('ownerGroupLdap',       'German',   'LDAP f&uuml;r Eigent&uuml;mergruppen');
+INSERT INTO txt VALUES ('ownerGroupLdap',       'English',  'LDAP for owner groups');
+INSERT INTO txt VALUES ('ownerGroupPattern',    'German',   'Namenskonvention f&uuml;r Egentt&uuml;mer-LDAP-Gruppen');
```
Comment (abarz722) [inline roles/database/files/sql/idempotent/fworch-texts.sql:2242]: typo

File: roles/database/files/sql/idempotent/fworch-texts.sql [hunk 2]
```diff
@@ -5164,6 +5170,14 @@ INSERT INTO txt VALUES ('H5638', 'English', 'Insert text for extra configuration
 INSERT INTO txt VALUES ('H5639', 'German',  'Hier wird der Meldungstext f&uuml;r Fehler im externen Beantragungsprozess eingetragen.');
 INSERT INTO txt VALUES ('H5639', 'English', 'Insert customized text for errors in requests to external ticket system.');
 
+INSERT INTO txt VALUES ('H5640', 'German',  'Auswahl des LDAP-Systems, in dem die Eigent&uuml;mergruppen gepflegt werden.');
```
Comment (abarz722) [inline roles/database/files/sql/idempotent/fworch-texts.sql:5173]: also include texts in HelpSettingsModelling.razor

Comment (tpurschke) [reply roles/database/files/sql/idempotent/fworch-texts.sql:5173]: thanks for finding this

File: roles/database/files/sql/idempotent/fworch-texts.sql (hunk 1 above)
Comment (tpurschke) [reply roles/database/files/sql/idempotent/fworch-texts.sql]: cheers - fixed

**Review by abarz722 — APPROVED** [review]

**Pull request merged**